packages/web-components/src/components/masthead/masthead-global-bar.ts

```diff
@@ -21,16 +22,70 @@ const { stablePrefix: ddsPrefix } = ddsSettings;
  */
 @customElement(`${ddsPrefix}-masthead-global-bar`)
 class DDSMastheadGlobalBar extends FocusMixin(LitElement) {
+  /**
+   * Media query on medium breakpoint.
+   */
+  private mqMedium;
+
   /**
    * The shadow slot this action bar should be in.
    */
   @property({ reflect: true })
   slot = 'profile';
 
+  /**
+   * Search bar opened flag.
+   */
+  @internalProperty()
+  private openedSearch = false;
+
+  /**
+   * Medium breakpoint flag.
+   */
+  @internalProperty()
+  private responsiveMd = false;
```

Comment:
The codebase has a coding convention to prefix private properties with `_`.

```suggestion
  private _responsiveMd = false;
```